recipes/musicbrainzngs/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "musicbrainzngs" %}
+{% set version = "0.7.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/musicbrainzngs-{{ version }}.tar.gz
+  sha256: ab1c0100fd0b305852e65f2ed4113c6de12e68afd55186987b8ed97e0f98e627
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=2.7
+  run:
+    - python >=2.7
+
+test:
+  imports:
+    - musicbrainzngs
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://python-musicbrainzngs.readthedocs.io/
+  summary: Python bindings for the MusicBrainz NGS and the Cover Art Archive webservices
+  license: BSD-3-Clause
```

Looks like it's actually:
```suggestion
  license: BSD-2-Clause AND MIT
```